src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/Sse.PlatformNotSupported.cs

```diff
@@ -24,550 +24,620 @@ public new abstract class X64 : X86Base.X64
 
         /// <summary>
         /// __m128 _mm_rsqrt_ss (__m128 a)
-        ///   RSQRTSS xmm, xmm/m32
+        ///    RSQRTSS xmm1,       xmm2/m32
+        ///   VRSQRTSS xmm1, xmm2, xmm3/m32
         /// </summary>
         public static Vector128<float> ReciprocalSqrtScalar(Vector128<float> value) { throw new PlatformNotSupportedException(); }
-
         /// <summary>
         /// __m128 _mm_rsqrt_ss (__m128 a, __m128 b)
-        ///   RSQRTSS xmm, xmm/m32
+        ///    RSQRTSS xmm1,       xmm2/m32
+        ///   VRSQRTSS xmm1, xmm2, xmm3/m32
         /// The above native signature does not exist. We provide this additional overload for consistency with the other scalar APIs.
         /// </summary>
         public static Vector128<float> ReciprocalSqrtScalar(Vector128<float> upper, Vector128<float> value) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m128 _mm_shuffle_ps (__m128 a,  __m128 b, unsigned int control)
-        ///   SHUFPS xmm, xmm/m128, imm8
+        ///    SHUFPS xmm1,               xmm2/m128,         imm8
+        ///   VSHUFPS xmm1,         xmm2, xmm3/m128,         imm8
+        ///   VSHUFPS xmm1 {k1}{z}, xmm2, xmm3/m128/m32bcst, imm8
         /// </summary>
         public static Vector128<float> Shuffle(Vector128<float> left, Vector128<float> right, [ConstantExpected] byte control) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m128 _mm_sqrt_ps (__m128 a)
-        ///   SQRTPS xmm, xmm/m128
+        ///    SQRTPS xmm1,         xmm2/m128
+        ///   VSQRTPS xmm1,         xmm2/m128
+        ///   VSQRTPS xmm1 {k1}{z}, xmm2/m128/m32bcst
         /// </summary>
         public static Vector128<float> Sqrt(Vector128<float> value) { throw new PlatformNotSupportedException(); }
 
         /// <summary>
         /// __m128 _mm_sqrt_ss (__m128 a)
-        ///   SQRTSS xmm, xmm/m32
+        ///    SQRTSS xmm1,               xmm2/m32
+        ///   VSQRTSS xmm1,         xmm2, xmm3/m32
+        ///   VSQRTSS xmm1 {k1}{z}, xmm2, xmm3/m32{er}
         /// </summary>
         public static Vector128<float> SqrtScalar(Vector128<float> value) { throw new PlatformNotSupportedException(); }
-
         /// <summary>
         /// __m128 _mm_sqrt_ss (__m128 a, __m128 b)
-        ///   SQRTSS xmm, xmm/m32
+        ///    SQRTSS xmm1,               xmm2/m32
+        ///   VSQRTSS xmm1,         xmm2, xmm3/m32
+        ///   VSQRTSS xmm1 {k1}{z}, xmm2, xmm3/m32{er}
         /// The above native signature does not exist. We provide this additional overload for consistency with the other scalar APIs.
         /// </summary>
         public static Vector128<float> SqrtScalar(Vector128<float> upper, Vector128<float> value) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// void _mm_storeu_ps (float* mem_addr, __m128 a)
+        ///    MOVUPS m128,         xmm1
```

Comment:
`xmm` instead of `xmm1` since there is no `xmm2`? (here and elsewhere)

Reply:
The standardized encoding the architecture manuals is to use `xmm1`, so we follow it here